packages/wallet/backend/src/rapyd/validation.ts

```diff
@@ -36,3 +36,14 @@ export const walletSchema = z.object({
     phone: z.string().optional()
   })
 })
+
```

Comment:
Oh i see you ddi add the schema, so maybe just use it in the controller? but the object should have `query` instead of 'body' for it to work